src/react-apps/applications/runtime/src/features/instantiate/containers/index.tsx

```diff
@@ -0,0 +1,102 @@
+import Grid from '@material-ui/core/Grid';
+import { createStyles, withStyles } from '@material-ui/core/styles';
+import * as React from 'react';
+import ContentLoader from 'react-content-loader';
+import { Redirect } from 'react-router-dom';
+import { IAltinnWindow } from 'src/types';
+import AltinnModal from '../../../../../shared/src/components/AltinnModal';
+import { get, post } from '../../../utils/networking';
+import Header from '../../header/containers';
+
+const styles = () => createStyles({
+  modal: {
+    boxShadow: null,
+    MozBoxShadow: null,
+    WebkitBoxShadow: null,
+  },
+  body: {
+    padding: 0,
+  },
+});
+
+function ServiceInfo(props) {
+  const [reportee, setReportee] = React.useState(null);
+  const [instanceId, setInstanceId] = React.useState(null);
+  const [instantiationError, setInstantiationError] = React.useState(null);
+
+  const fetchReportee = async () => {
+    const url: string = `${window.location.origin}/runtime/api/v1/profile/user`;
```

Comment:
@TheTechArch : Will this path work in the deployed cluster?

Reply:
If it doesn't work, we could just add:
```
// indexOf returns -1 if it doesn't find
if (window.location.origin.indexOf('altinn.studio') < 0) {
  ...
}
```

Reply:
That could be a temporary solution until Studio and Apps have the same paths.

Reply:
@matsgm  same comment as above.  We should create a issue to update GITEA in Altinn Studio to use a path and start using runtime as default app behind nginx

Reply:
@jezpoz : #2029 should fix the path to Gitea in Altinn Studio, so no logic is necessary and no ```/runtime/``` path in Studio or Apps